docs/plone_training_config/instructions_plone5.md

```diff
@@ -58,12 +58,12 @@ sudo apt-get -y upgrade
 Then, you need to install the following packages:
 
 ```shell
-sudo apt-get install python3.7-dev python3.7-tk python3.7-venv build-essential libssl-dev libxml2-dev libxslt1-dev libbz2-dev libjpeg62-dev
+sudo apt-get install python3.8-dev python3.8-tk python3.8-venv build-essential libssl-dev libxml2-dev libxslt1-dev libbz2-dev libjpeg62-dev
```

Comment:
Why not go up to python3.9?

Reply:
Yeah I thought about that.... 3.10 has just come out too. Should I go with that?

Reply:
all right, I'll just make it 3.9 and be done with it for today :)

Reply:
I have a personal rule: "I shall not publish anything that I have not personally tried." IOW, don't make up 💩 .

If you have tried it and it works, then yes. Else, stick with what you know.